htmlPlugin.js

```diff
@@ -0,0 +1,40 @@
+const { appInfo: { loki } } = require('./package.json');
+
+const resourceRootUrn = `urn:com:${loki.cloudCodeName}:${loki.appCodeName}:app:pages:${loki.pageCodeName}!`;
+/**
+ * @param {string} htmlstring An HTML string
+ * @param {string} filename A bundle file name to replace with "${loki.web.resourceUrl('[b]')}""
+ */
+function replacePathWithLokiResource(htmlstring, filename) {
+  const regexp = new RegExp(
+    `(=")(?:)[^=]*(?:)(${filename})(?:)[^=]*(")`,
+    'g',
+  );
+  return htmlstring.replace(
+    regexp,
+    `$1\${loki.web.resourceUrl('${resourceRootUrn}$2')}$3`,
+  );
+}
+const htmlPlugin = () => {
+  /** @type {import('vite').Plugin} */
+  const def = {
```

Comment:
Could we name this something more descriptive for legibility?

Reply:
You mean make it meaningful to anyone but me? Of course! I copied that from an example plugin on Vite's site. So I'm not emotionally committed to it.

Reply:
Done